src/ray/core_worker/core_worker.cc

```diff
@@ -719,4 +719,29 @@ Status CoreWorker::BuildArgsForExecutor(const TaskSpecification &task,
   return status;
 }
 
+Status CoreWorker::ActorLanguage(const ActorID &actor_id, Language *language) const {
```

Comment:
These methods should not be exposed on the `CoreWorker`. You can instead call `CoreWorker::GetActorHandle`, then call the relevant method on the returned actor handle.

Reply:
OK, updated.